apps/src/templates/sectionProgressV2/progress-table-legend.module.scss

```diff
@@ -1,7 +1,7 @@
 @import 'color';
 
 .iconKey {
-  margin-bottom: 40px;
+  margin-bottom: 24px;
```

Comment:
This accounts for the margin at the bottom of each box.  I thought about using media queries, but this seemed like a simpler solution.